CHANGELOG.md

```diff
@@ -1,9 +1,40 @@
 # Optimizely Python SDK Changelog
 
-## Unreleased
+## 5.0.0-beta
+Apr 27th, 2023
 
-### Breaking Changes:
-* `PollingConfigManager` now requires `sdk_key` even when providing a url. ([#413](https://github.com/optimizely/python-sdk/pull/413))
+### New Features  
+
+* The 4.0-beta release introduces a new primary feature, [Advanced Audience Targeting]( https://docs.developers.optimizely.com/feature-experimentation/docs/optimizely-data-platform-advanced-audience-targeting) enabled through integration with [Optimizely Data Platform (ODP)](https://docs.developers.optimizely.com/optimizely-data-platform/docs) ([#395](https://github.com/optimizely/python-sdk/pull/395), [#398](https://github.com/optimizely/python-sdk/pull/398), [#402](https://github.com/optimizely/python-sdk/pull/402), [#403](https://github.com/optimizely/python-sdk/pull/403), [#405](https://github.com/optimizely/python-sdk/pull/405)). 
+* You can use ODP, a high-performance [Customer Data Platform (CDP)]( https://www.optimizely.com/optimization-glossary/customer-data-platform/), to easily create complex real-time segments (RTS) using first-party and 50+ third-party data sources out of the box. You can create custom schemas that support the user attributes important for your business, and stitch together user behavior done on different devices to better understand and target your customers for personalized user experiences. ODP can be used as a single source of truth for these segments in any Optimizely or 3rd party tool.
+* With ODP accounts integrated into Optimizely projects, you can build audiences using segments pre-defined in ODP. The SDK will fetch the segments for given users and make decisions using the segments. For access to ODP audience targeting in your Feature Experimentation account, please contact your Optimizely Customer Success Manager. 
```

Comment:
Let's remove these bullets at the top level to be consistent with other SDKs (see https://github.com/optimizely/swift-sdk/blob/d5e190b0aacc423c98ea133eb6d386eeb2bb019e/CHANGELOG.md)